tutorials/Tutorial_For_QA_Registration.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QA REGISTRATION\n",
+    "Give a visual result for regisration part to help user check results of each step. The graph includes three dimensions(sagital slice, coronal slice and axial slice) and uses pink and green to indicate the registered output and reference image."
```

Comment:
sagittal, not sagital

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/7b32b3d114ee8d7a65060fb0dc1fefa8393cb052